QA.py

```diff
@@ -49,13 +51,13 @@ def __init__(self, q_format, db_query, format_answer):
         self.db_query = db_query
         self.format_answer = format_answer
```

Comment:
For example, the monkeytype annotations of these function arguments mismatch the docstring _Args_ in this `__init__`